learn-ocaml.el

```diff
@@ -521,6 +521,7 @@ Argument SECRET may be needed by the server."
             nil
             server
             (lambda (_)
+              (learn-ocaml-change-token)
               (message-box "Server changed succesfully")
```

Comment:
@LouisAyroles don't you think these two lines should be swapped?
first, the message on server-change,
then the question(s) about tokens?

Reply:
OK, so not merging yet and putting the PR in draft mode; feel free to get out of the draft mode when you'll think it's ready :)